frontend/src/metabase/query_builder/containers/QueryBuilder.unit.spec.tsx

```diff
@@ -1007,12 +1083,11 @@ describe("QueryBuilder", () => {
       it("does not show custom warning modal when saving edited question", async () => {
         const { history } = await setup({
           card: TEST_NATIVE_CARD,
-          initialRoute: "/",
+          initialRoute: `/question/${TEST_NATIVE_CARD.id}`,
```

Comment:
In order for `history.goBack()` to work, there needs to be at least 1 previous route in the history.
For that reason almost all tests using `history.goBack()` will have `initialRoute: "/"`.

In this test I was able to skip that, because there will be a redirect after saving the question (from `/question/<id>` to `/question/<slug>`, so there will be an entry in history that we can go back to.

We still could have `initialRoute: "/"` for consistency here, but `history.goBack()` wouldn't go back to `"/"` - so it feels redundant to use it here.